misc/10-aws.conflist

```diff
@@ -1,4 +1,5 @@
 {
+  "cniVersion": "0.3.0",
```

Comment:
Thanks for adding this, just a minor nit, it should be:  `"0.3.1"`.

It's just a [small spelling fix in the spec from v0.3.0](https://github.com/containernetworking/cni/releases/tag/spec-v0.3.1), but we do return `ips` and not `ip`.

https://github.com/aws/amazon-vpc-cni-k8s/blob/566b0d44701a6c1bcd8e2c4022cc6cfc7b7c0c59/plugins/routed-eni/cni.go#L213

Reply:
Changed it, and tested.

Reply:
Was just about to bump the version in my repo then push the changes to my repo & re-submit 

Thanks !